WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -55,6 +61,12 @@ - (id)initWithStyle:(UITableViewStyle)style
                                                                       target:nil
                                                                       action:nil];
         [self.navigationItem setBackBarButtonItem:backButton];
+        
+        UIBarButtonItem *searchButton = [[UIBarButtonItem alloc] initWithImage:[UIImage imageNamed:@"icon-post-search"]
+                                                                         style:UIBarButtonItemStylePlain
+                                                                        target:self
+                                                                        action:@selector(toggleSearch)];
+        [self.navigationItem setRightBarButtonItem:searchButton];
```

Comment:
It might be nice to go ahead and clean up the self references in `init`. They were already there, but we're also making a new change.  Doing the assignment directly to the ivar avoids some edge cases involving accessors when an object isn't fully initialized.